.github/workflows/pythonpackage.yml

```diff
@@ -24,7 +24,6 @@ jobs:
         python -m pip install --upgrade pip
         pip cache purge
         pip install .[dev]
-        pip install lib/kql lib/kibana
```

Comment:
If needed, we can also move this line up instead of removing it so that it installs from the local environment.